dashboard/templates/dashboard/500.html

```diff
@@ -0,0 +1,20 @@
+{% extends "public_website/base.html" %}
+
+{% load i18n %}
+{% load to_language_name %}
+
+{% block title %} {% trans 'Internal Server Error' %} | {% trans 'Sawaliram' %} {% endblock %}
+
+{% load static %}
+
+{% block navbar %} {% endblock %}
+
+{% block content %}
+
+<div class="error-page-wrapper text-center">
+    <img src="{% static 'error_icons/coding.png' %}" alt="{% trans 'Internal Server Error' %}">
+    <h3> {% trans 'Sorry something has gone wrong on the server!' %}</h3>
```

Comment:
I was wondering whether there is any benefit in not hardcoding the text in the template.